sdk/typescript/test/unit/cryptography/ed25519-keypair.test.ts

```diff
@@ -62,4 +62,42 @@ describe('ed25519-keypair', () => {
     );
     expect(isValid).toBeTruthy();
   });
+
+  it('derive ed25519 keypair from path and mnemonics', () => {
+    // Test case generated against rust: /sui/crates/sui/src/unit_tests/keytool_tests.rs#L149
+    const keypair = Ed25519Keypair.deriveKeypair(
+      `m/44'/784'/0'/0'/0'`,
+      'result crisp session latin must fruit genuine question prevent start coconut brave speak student dismiss'
```

Comment:
As we are reusing this, just set it to a const.

Reply:
oh, I was referring to the mnemonic string reuse in the tests, not the derived key :)